settings.gradle.kts

```diff
@@ -1,14 +1,28 @@
 pluginManagement {
+    val agp = "7.0.0-alpha09"
+    val kotlin = "1.4.20"
```

Comment:
I've lowered kotlin version because of crashing static code analysis step: https://youtrack.jetbrains.com/issue/KT-44735 

#7